lib/timelines/models/concerns/audit_trail.rb

```diff
@@ -0,0 +1,21 @@
+module Timelines
+  class AuditTrail
+    include ActiveModel::Model
+    include ActiveModel::Attributes
+    include ActiveModel::Validations
+
+    attribute :resource
+    attribute :reverse
+    attribute :events
+
+    def events
+      reverse ? reverse_events : resource.events
```

Comment:
This returns a list of `Timelines::Event` records and we have `.audit_trail` returning both the `resource` and the events. Do you think we still need a `::Timelines::AuditTrail::Event` class?